application/modules/invoices/controllers/Ajax.php

```diff
@@ -435,9 +435,16 @@ public function create_credit()
             $this->mdl_invoices->where('invoice_id', $target_id);
             $this->mdl_invoices->update('ip_invoices', ['creditinvoice_parent_id' => $source_id]);
 
+            // Change invoice terms to credit note terms
+            $this->mdl_invoices->where('invoice_id', $target_id);
+            $this->mdl_invoices->update('ip_invoices', ['invoice_terms' => get_setting('default_creditnote_terms')]);
+
             $this->mdl_invoices->where('invoice_id', $target_id);
             $this->mdl_invoices->update('ip_invoice_amounts', ['invoice_sign' => '-1']);
 
+            // Update the invoice date created (today) and due date
```

Comment:
Is it right for a credit note to have a due date? The function is also setting a due date.

Reply:
_I know..._
I see a **'credit note due date'** in IP dashboard as an indication that I still need to take action and I find it useful because I can immediately see that there is still an (_expired?_) refund to be made to my customer.
As a result, the (_credit note_) due date in the **InvoicePlane template** must of course also be adjusted.